pkg/metrics/metrics.go

```diff
@@ -104,14 +104,14 @@ var End2EndEventTimeHistogram = prometheus.NewHistogramVec(prometheus.HistogramO
 	Namespace: "gravity",
 	Name:      "event_time_latency",
 	Help:      "Latency of end to end event time in seconds.",
-	Buckets:   prometheus.ExponentialBuckets(0.0005, 2, 22), // ~ 17min
+	Buckets:   prometheus.ExponentialBuckets(0.0005, 2, 30), // ~ 74h
```

Comment:
`(0.0005, 2, 30)` would use too much resources I think, we can consider using a larger `factor` instead.

Reply:
PTAL